src/common/messages.js

```diff
@@ -188,6 +200,128 @@ export const messages = {
   fillOptionalInformations: {
     defaultMessage: 'Fill informations (optional)'
   },
+  selectTableReset: {
+    defaultMessage: 'Reset'
+  },
+
+  // --- Dates/Scheduler ---
+
+  schedulerMonth: {
+    defaultMessage: 'Month'
+  },
+  schedulerEveryMonth: {
+    defaultMessage: 'Every month'
+  },
+  schedulerEachSelectedMonth: {
+    defaultMessage: 'Each selected month'
+  },
+  schedulerMonthDay: {
+    defaultMessage: 'Day of the month'
+  },
+  schedulerEveryMonthDay: {
+    defaultMessage: 'Every day'
+  },
+  schedulerEachSelectedMonthDay: {
+    defaultMessage: 'Each selected day'
+  },
+  schedulerWeekDay: {
+    defaultMessage: 'Day of the week'
+  },
+  schedulerEveryWeekDay: {
+    defaultMessage: 'Every day'
+  },
+  schedulerEachSelectedWeekDay: {
+    defaultMessage: 'Each selected day'
+  },
+  schedulerHour: {
+    defaultMessage: 'Hour'
+  },
+  schedulerEveryHour: {
+    defaultMessage: 'Every hour'
+  },
+  schedulerEveryNHour: {
+    defaultMessage: 'Every N hour'
+  },
+  schedulerEachSelectedHour: {
+    defaultMessage: 'Each selected hour'
+  },
+  schedulerMinute: {
+    defaultMessage: 'Minute'
+  },
+  schedulerEveryMinute: {
+    defaultMessage: 'Every minute'
+  },
+  schedulerEveryNMinute: {
+    defaultMessage: 'Every N minute'
+  },
+  schedulerEachSelectedMinute: {
+    defaultMessage: 'Each selected minute'
+  },
+  schedulerReset: {
+    defaultMessage: 'Reset'
+  },
+  january: {
```

Comment:
Create [an issue in react-intl](https://github.com/yahoo/react-intl/issues) explaining the problem.